daemon/daemon_windows.go

```diff
@@ -264,6 +264,35 @@ func (daemon *Daemon) initNetworkController(activeSandboxes map[string]interface
 		}
 
 		if !found {
+			// non-default nat networks should be re-created if missing from HNS
+			if v.Type() == "nat" && v.Name() != "nat" {
+				_, _, v4Conf, v6Conf := v.Info().IpamConfig()
+				netOption := map[string]string{}
+				for k, v := range v.Info().DriverOptions() {
+					if k != winlibnetwork.NetworkName && k != winlibnetwork.HNSID {
+						netOption[k] = v
+					}
+				}
+				name := v.Name()
+				id := v.ID()
+
+				err = v.Delete()
```

Comment:
Just to be sure; would the `ingress` network also be in this list? If so, do we need to pass `NetworkDeleteOptionRemoveLB` as option? https://github.com/moby/moby/blob/5de77049db9d6aed893f086fc672cb8df4532650/libnetwork/network.go#L915

Reply:
Oh, I guess the `ingress` network wouldn't be `v.Type() == "nat"` ? If that's so, ignore my comment

Reply:
`ingress` is `v.Type() == "overlay"` so this does not affect those.

Reply:
Thanks! Yes, I realised that after writing my first comment; thanks for confirming 👍